apps/admin-server/src/pages/projects/create.tsx

```diff
@@ -26,18 +26,47 @@ const formSchema = z.object({
   }),
 });
 
+const importFormSchema = z.object({
+  importedProjectName: z.string().min(6, {
+    message: 'Het project moet minimaal uit zes karakters bestaan!'
+  })
+})
+
 export default function CreateProject() {
-  const { createProject } = useProject()
+  const { createProject, importProject } = useProject()
+  const [file, setFile] = React.useState("")
 
   const form = useForm<z.infer<typeof formSchema>>({
     resolver: zodResolver<any>(formSchema),
     defaultValues: {},
   });
 
+  const importForm = useForm<z.infer<typeof importFormSchema>>({
+    resolver: zodResolver<any>(importFormSchema),
+    defaultValues: {}
+  })
+
+  function handleChange(e: any) {
+    const fileReader = new FileReader();
+    fileReader.readAsText(e.target.files[0], "UTF-8");
+    fileReader.onload = e => {
+      setFile(e.target?.result)
+    }
+  }
+
   async function onSubmit(values: z.infer<typeof formSchema>) {
     const project = await createProject(values.projectName);
     if (project) {
-      toast.success('Widget aangemaakt!');
+      toast.success('Project aangemaakt!');
+      router.push(`/projects/${project.id}/widgets`);
+    }
+  }
+
+  async function onImport(values: z.infer<typeof importFormSchema>) {
+    const data = JSON.parse(file)
```

Comment:
Wat als ik ervoor kies om een image up te loaden of een csv bestand, crasht het dan? Omdat het gaat om uploaden zou ik er een try catch omheen doen